README.md

```diff
@@ -1,15 +1,41 @@
+![image](https://github.com/scale-it/algo-builder/blob/master/media/logo.svg) 
```

Comment:
you should use a cdn link: https://gitcdn.link/

Reply:
okay